Tasks/CmdLineV2/package.json

```diff
@@ -18,6 +18,11 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "uuid": "^3.0.1",
+    "@types/mocha": "^8.0.3",
+    "@types/node": "^6.0.0",
     "azure-pipelines-task-lib": "^2.9.3"
+  },
+  "devDependencies": {
+    "typescript": "4.0.2"
```

Comment:
This is how tasks indicate they want a newer TS version.

I would prefer having something in the `make.json` to specify this, however I could not figure out a simple way to download another TS version along with 2.3.4 from the root project. Any ideas?

Reply:
There's not a great way to do this AFAIK, I think this approach probably makes sense?

Reply:
Okay cool. Just wanted to make sure